traefik.docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
```

Comment:
Is this file needed? This block appears in the docker-compose.yml already

Reply:
It is needed, docker won't run without it.